website/content/docs/upgrading/deduplication/different-case.mdx

```diff
@@ -0,0 +1,184 @@
+
+- You have [identified the duplicate entity aliases](/vault/docs/upgrading/identity-deduplication). 
+- You have admin permission on the relevant Vault server or cluster.
+
+</Tip>
+
+
+## Why entity alias duplicates happen
+
+Historical bugs in Vault allowed authN mounts to create duplicate entries of the
+same entity alias if the aliases used difference cases. For example, if an
+external LDAP system returns unnormalized usernames or changes the normalization
+scheme over time. By default, Vault uses case-insensitive matching for the
+usernames and considers `username` and `uSeRnAmE` to be the same alias. But,
+past bugs have left some users with different-case duplicates in storage. To
+avoid the security impact of incorrectly merging identities that do not
+represent the same user, Vault tries to honor past behavior by switching the
+identity engine to use case-sensitive matching mode.
+But case-sensitive matching 
+This mode is not well supported however  
+as it deviates from Vault's typical assumptions. Remaining in this mode can lead to unexpected behavior. 
+
+
+## Example server log
+
+```text
+[WARN]  identity: 2 different-case entity alias duplicates found (potential security risk)
+[WARN]  identity: entity-alias "alias-case" with mount accessor "auth_userpass_34aca7ec" duplicates 1 others: id=df3568a4-3b65-4104-9481-1129ecbed72f canonical_id=5f013d99-a6c7-9a00-6ad5-4ad724b14f60 force_deduplication="would merge into entity 7da76b0d-fe9b-a125-3362-2a8ff055dcf8"
+[WARN]  identity: entity-alias "alias-cAsE" with mount accessor "auth_userpass_34aca7ec" duplicates 1 others: id=2992253b-1e99-4e47-b6f9-afb0c7cedf7a canonical_id=7da76b0d-fe9b-a125-3362-2a8ff055dcf8 force_deduplication="would merge others into this entity"
+[WARN]  identity: entity-alias "alias-case" with mount accessor "auth_userpass_a555989a" duplicates 1 others: id=0e4bd46e-a868-4dd4-a34a-cb73f097e3a5 canonical_id=1ce06951-e2fd-0923-8ae0-a0e2c6c2378b force_deduplication="would merge into entity 37f41fcf-5e15-f13f-248d-9fa8405b1ecd"
+[WARN]  identity: entity-alias "alias-cAsE" with mount accessor "auth_userpass_a555989a" duplicates 1 others: id=6cafe546-665c-4bd0-a4fd-5e699ba413c1 canonical_id=37f41fcf-5e15-f13f-248d-9fa8405b1ecd force_deduplication="would merge others into this entity"
+[WARN]  identity: end of different-case entity-alias duplicates
+```
+
+The duplicate alias in the example log are distinct, despite have the same name,
+because they come from different authN mounts. When resolving duplicates,
+consider the set of duplicate names from each authN mount separately.
+
+Before resolving different-case duplicates, you must determine if the case
+difference is a **mergeable duplicate** or **unmergeable duplicate**:
+
+- For mergeable duplicates, the case difference is **unintentional** and the
+  duplicate entries represent the same logical entity. You should deduplicate
+  true duplicates to reflect the expected default Vault behavior.
```

Comment:
```suggestion
- For mergeable duplicates, the case difference is **unintentional** and the
  duplicate entries represent the same logical entity. It is safe to ignore these 
  and allow them to be merged automatically when the `force-identity-deduplication` 
  flag is activated.
```
Unsure if this is needed here as we say it below, but it feels strange to me to say "you should deduplicate" when the instructions to fix below are "ignore these". The instruction here sounds like "action required" when it's really "no action required"?